web/datapipeline/clusters_projector.go

```diff
@@ -79,17 +86,17 @@ func detectClusterType(cluster *cluster.Cluster) string {
 
 	switch {
 	case hasSapHanaTopology && hasSAPHana:
-		return models.ClusterTypeScaleUp
+		return models.ClusterTypeHANAScaleUp
 	case hasSapHanaTopology && hasSAPHanaController:
-		return models.ClusterTypeScaleOut
+		return models.ClusterTypeHANAScaleOut
 	default:
 		return models.ClusterTypeUnknown
 	}
 }
 
-// getHanaSIDs returns the SIDs of the cluster
+// getClusterSIDs returns the SIDs of the cluster
```

Comment:
Being honest, I think there was some misunderstanding here with this method.
As far as I know, the cluster only manages one principal SID (for scale up or scale out), and a possible cost-optimized SID.
So, 1 is always present, and the 2nd one only in cost-optimized scenarios.
As far as I know, scale out deployments only have 1 SID too.

I would go more for `SID` and `CostOptimizedSID` columns.

Reply:
Alright, but since we don't have the cost optimized i would just refactor to use one SID (get the first) and then we can iterate on this.

Reply:
https://github.com/trento-project/trento/issues/493